docs/sdk/serverless-signing/conditional-signing.md

```diff
@@ -5,123 +5,56 @@ import TabItem from '@theme/TabItem';
 
 # Conditional Signing
 
-## Prerequisites
-
-- Familiarity with JavaScript
-- Basic understanding of [serverless signing](../serverless-signing/quick-start.md)
-
 ## Overview
-Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs. This system can be used to uphold the integrity of data on the open web, in its function as a decentralized notary.
+Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. This means that you can easily check any on-chain condition inside a Lit Action, which can be useful for generating proofs.
 
 The below example will check if the user has at least 1 Wei on Ethereum, only returning a signature if they do.
 
-:::note
-`toSign` data is required to be in 32 byte format. 
+## Prerequisites
 
-The `ethers.utils.arrayify(ethers.utils.keccak256(...)` can be used to convert the `toSign` data to the correct format.
-:::
+- Knowlege of [SessionSigs](../authentication/session-sigs/intro)
+- Knowledge of how to [generate an AuthSig](../migrations/6.0.0.md#generate-an-authsig)
```

Comment:
isn't the auth sig portion covered in the session sig docs already?

Reply:
Somewhat. It mentions AuthSigs, but it is more in the context of using `authNeededCallback` to generate the SessionSigs.

Doing the `conditional-signing` requires you to use the `checkConditions` function from the Lit Actions SDK, which means generating an AuthSig outside of SessionSigs. With this in mind, do you think it should be removed from here?

There was a user on Discord a couple days ago trying to do conditional-signing, and ran into this issue. I think decrypting and checking conditions in the Lit Actions SDK still relies on AuthSigs and not SessionSigs.

Reply:
oh got it okay, so it is indeed a separate process. In that case, i think you made the right call